hiddenlayer/sdk/services/model_scan.py

```diff
@@ -132,10 +138,9 @@ def scan_file(
                 scan_results = self.get_scan_results(model_name=model_name)
                 print(f"{file_path.name} scan status: {scan_results.status}")
 
-        scan_results = ScanResults.from_scanresultsv2(scan_results_v2=scan_results)
         scan_results.file_name = file_path.name
         scan_results.file_path = str(file_path)
-        scan_results.sensor_id = sensor.sensor_id
+        scan_results.model_id = sensor.sensor_id
```

Comment:
sensor id and model id aren't the same thing.  sensor id is more analogous to a model version id

Reply:
Looking at the code, I think we don't need to set anything here.  I'll drop this line.  Thanks for the feedback.